themes/sexy/sexy.theme.bash

```diff
@@ -34,9 +38,16 @@ parse_git_dirty () {
 parse_git_branch () {
   git branch --no-color 2> /dev/null | sed -e '/^[^*]/d' -e "s/* \(.*\)/\1$(parse_git_dirty)/"
 }
+env_prompt () {
+  echo -e "($(virtualenv_prompt)$(condaenv_prompt))"
+}
 
 function prompt_command() {
   PS1="\[${BOLD}${MAGENTA}\]\u \[$WHITE\]at \[$ORANGE\]\h \[$WHITE\]in \[$GREEN\]\w\[$WHITE\]\$([[ -n \$(git branch 2> /dev/null) ]] && echo \" on \")\[$PURPLE\]\$(parse_git_branch)\[$WHITE\]\n\$ \[$RESET\]"
+
+  if [ "$SEXY_WITH_PYTHON" = true ] ; then
```

Comment:
A couple of things:
* It would be great if you could indicate that this variable relates to the sexy _theme_. Maybe rename it to `SEXY_THEME_SHOW_PYTHON`?
* Please consider initializing the variable at the top of the file, similar to how other themes handle this, e.g. `SEXY_THEME_SHOW_PYTHON="${SEXY_THEME_SHOW_PYTHON:=false}" - this way, the variable always has a default value (`false`).

Reply:
Thanks for the great suggestion. I implemented both, and deleted the original additional theme.